lib/Wikia/src/Service/User/Attributes/AttributeKeyValueService.php

```diff
@@ -27,7 +28,13 @@ function __construct( AttributePersistence $persistenceAdapter ) {
 		$this->persistenceAdapter = $persistenceAdapter;
 	}
 
-	public function set( $userId, $attribute ) {
+	/**
+	 * @param int $userId
+	 * @param Attribute $attribute
+	 * @return true
```

Comment:
Before it was just true or exception, but I think there's a case where it should return false. I'll update.